.github/workflows/go.yml

```diff
@@ -0,0 +1,131 @@
+name: Build and Test
+
+on:
+  push:
+    branches:
```

Comment:
Since this is "all branches" I think we can just leave this out. Also on the `pull_request` section.

Reply:
yup, done